app/src/main/java/com/concordium/wallet/util/DoubleExtensions.kt

```diff
@@ -1,7 +1,7 @@
 package com.concordium.wallet.util
 
-import kotlin.math.floor
-import kotlin.math.pow
+import java.math.BigDecimal
+import java.math.RoundingMode
 
-fun Double.truncateDecimals(decimalPoint: Int): Double =
-    floor(this * 10.0.pow(decimalPoint)) / 10.0.pow(decimalPoint)
\ No newline at end of file
+fun Double.roundToDecimalPlace(numberOfDecimalPlaces: Int) =
+    BigDecimal(this).setScale(numberOfDecimalPlaces, RoundingMode.DOWN).toDouble()
```

Comment:
Now I get 57.559 -> 57.557, I guess because we always "round down". :thinking: 
Is there an issue using `RoundingMode.HALF_DOWN`?